jobs/comms_invite.nomad

```diff
@@ -0,0 +1,31 @@
+job "comms-invite" {
+  datacenters = ["dc1"]
+
+  group "comms-invite" {
+    count = 1
+
+    task "comms-invite" {
+      driver = "docker"
+
+      config {
+        image = "hs3city/comms-invite:latest"
```

Comment:
Would it be ok to pin the image tag?
I know I'm using `latest` in some other jobs - but let's try to be kosher ;)

Reply:
We can do this when a versioned image will be pushed. This app is so small it shouldn't be a problem until some real gitops automation kicks in.